### PR TITLE
Add osquery POC package

### PR DESCRIPTION
**Pull request by tsg**
This adds a package for the POC we are doing on an osquery integration. We don't intend to merge this anytime soon, but I opened the PR in draft mode so that it is easier to find.

### CODE REVIEW

File: packages/osquery/0.1.0/img/logo_osquery.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="2500" height="2490" viewBox="0 0 256 255" xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMidYMid"><path d="M255.215.258v63.736l-63.605 63.605v-63.87L255.215.259" fill="#A596FF"/><path d="M128.006.258v63.736l63.605 63.605v-63.87L128.006.259"/><path d="M255.346 254.804H191.61l-63.605-63.605h63.867l63.474 63.605" fill="#A596FF"/><path d="M255.346 127.595H191.61L128.005 191.2h63.867l63.474-63.605"/><path d="M.801 254.936V191.2l63.605-63.605v63.869L.801 254.936" fill="#A596FF"/><path d="M128.01 254.936V191.2l-63.605-63.605v63.869l63.604 63.473"/><path d="M.672.386h63.736l63.605 63.605H64.145L.672.386" fill="#A596FF"/><path d="M.672 127.594h63.736l63.605-63.605H64.145L.672 127.594"/></svg>
```
Comment (ruflin) [inline packages/osquery/0.1.0/img/logo_osquery.svg:1]: Nice, even an icon :-)

**Comment by ruflin** [comment]
We have a few example and test packages here: https://github.com/elastic/package-registry/tree/master/testdata/package I wonder if we could add it there as an easy place to find it?

**Comment by mtojek** [comment]
@ruflin shouldn't it be open against integrations repository? or in a dedicated one like endpoint?

**Comment by ruflin** [comment]
@mtojek yes, probably. This one here is pretty special as we don't even have the binary for it yet, at least not shipped.

As long as it doesn't get merged, I'm good with almost all places :-)

**Comment by ruflin** [comment]
@tsg Currently trying to clean up `package-storage` a bit. Any chance to close this PR and open it against `integrations` repo instead?

**Comment by tsg** [comment]
Yep, happy to close this one. Pinging @aleksmaus here for visibility. Aleks, do you have a newer version of this to open against the `integrations` repo? If not, I can redo this one there.

**Comment by aleksmaus** [comment]
@tsg It's been awhile. Need to remember where was the latest one, that I used for demo and check if it's still applicable (working). Will dig around today.

**Pull request closed without merging**